setup.py

```diff
@@ -277,7 +277,6 @@ def write_version(filename: str = os.path.join(*["airflow", "git_version"])):
 # DEPENDENCIES_EPOCH_NUMBER in the Dockerfile
 ############################################################################################################
 devel = [
-    'astroid~=2.2.5',  # to be removed after pylint solves this: https://github.com/PyCQA/pylint/issues/3123
```

Comment:
I think we can also remove the pylint limit below (same reason) and then we can finally disable module-level docstrings :)

Reply:
I wanted to create a separate PR for that due to a huge amount of changes when upgrading pylint to >= 2.4

Reply:
OK then :).